variables.tf

```diff
@@ -1,49 +1,23 @@
-variable "aws_region" {
-  description = "AWS region."
-  type        = "string"
-}
-
-variable "environment" {
-  description = "A name that identifies the environment, will used as prefix and for tagging."
-  type        = "string"
-}
-
-variable "vpc_id" {
-  description = "The VPC that is used for the instances."
-  type        = "string"
-}
-
-variable "subnet_id_runners" {
-  description = "Subnet used to hosts the docker-machine runners."
-  type        = "string"
-}
-
 variable "availability_zone_runners" {
   description = "Availability zone used to host the docker-machine runners."
   type        = "string"
   default     = "a"
 }
 
-variable "availability_zone_proxy" {
-  description = "Availability zone used to host the proxy."
-  type        = "string"
-  default     = "a"
-}
-
-variable "subnet_id_gitlab_runner" {
-  description = "Subnet used for hosting the gitlab-runner."
+variable "aws_region" {
+  description = "AWS region."
   type        = "string"
 }
 
-variable "instance_type" {
-  description = "Instance type used for the gitlab-runner."
+variable "cache_user" {
```

Comment:
This variable has no any references in code

Reply:
variable "instance_type" is used in runner-from-prebacked-ami.tf

Reply:
```
tf-module-aws-gitlab-runner(develop) » grep -ri 'cache_user' *                                                                                                               

README.md:| cache_user                    | User name of the user to create to write and read to the s3 cache.                                                  | string |   `cache_user`   |    no    |
variables.tf:variable "cache_user" {
variables.tf:  default     = "cache_user"
```

Reply:
Fixed